src/masternode/meta.cpp

```diff
@@ -10,6 +10,13 @@
 
 const std::string MasternodeMetaStore::SERIALIZATION_VERSION_STRING = "CMasternodeMetaMan-Version-5";
 
+static constexpr int MASTERNODE_MAX_FAILED_OUTBOUND_ATTEMPTS{5};
+static constexpr int MASTERNODE_MAX_MIXING_TXES{5};
+
+namespace {
+static const CMasternodeMetaInfo default_meta_info_meta_info{};
```

Comment:
that's a weird name... why not
```suggestion
static const CMasternodeMetaInfo default_meta_info{};
```
?